fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/network/method/message/data/federation/TokensExchangeTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.github.dedis.popstellar.model.network.method.message.data.federation
+
+import com.github.dedis.popstellar.model.network.method.message.data.Action
+import com.github.dedis.popstellar.model.network.method.message.data.Objects
+import com.github.dedis.popstellar.model.objects.Lao
+import com.github.dedis.popstellar.testutils.Base64DataUtils
+import org.junit.Assert
+import org.junit.Test
+import java.time.Instant
+
+class TokensExchangeTest {
```

Comment:
don't forget to add constructors succeed and failing tests as in the other class